src/PIL/ImageChops.py

```diff
@@ -244,8 +238,10 @@ def subtract_modulo(image1, image2):
 
 
 def logical_and(image1, image2):
-    """Logical AND between two images. At least one of the images must have
-    mode "1".
+    """Logical AND between two images.
+
+    Both of the images must have mode "1". For an AND in RGB mode, use a
+    multiply() by a black-and-white mask.
```

Comment:
```suggestion
    Both of the images must have mode "1". If you would like to perform a
    logical AND on an image with a mode other than "1", try
    :py:meth:`~PIL.ImageChops.multiply` instead, using a black-and-white mask
    as the second image.
```

Suggestion. This advice isn't exclusive to RGB.

The ':py:meth' part is a way to link the method in the docs. For an example, see the [close method docstring](https://github.com/python-pillow/Pillow/blob/a28b2ea11c5c7e63adb9b29d136d412c862f7328/src/PIL/Image.py#L590) and the [way it is rendered](https://pillow.readthedocs.io/en/stable/reference/Image.html#PIL.Image.Image.close).